prime-dotnet-webapi/Controllers/BannersController.cs

```diff
@@ -0,0 +1,144 @@
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+using Prime.Auth;
+using Prime.Models;
+using Prime.Models.Api;
+using Prime.Services;
+using Prime.ViewModels;
+
+namespace Prime.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/[controller]")]
+    [ApiController]
+    public class BannersController : ControllerBase
+    {
+        private readonly IBannerService _bannerService;
+        private readonly IMetabaseService _metabaseService;
+        private readonly IAdminService _adminService;
+
+        public BannersController(IBannerService bannerService, IMetabaseService metabaseService, IAdminService adminService)
+        {
+            _bannerService = bannerService;
+            _metabaseService = metabaseService;
+            _adminService = adminService;
+        }
+
+        // PUT: api/Banners/enrolment-landing
+        /// <summary>
+        /// Create/update enrollee landing banner
+        /// </summary>
+        /// <param name="viewModel"></param>
+        [HttpPut("enrolment-landing", Name = nameof(CreateOrUpdateEnrolleeLandingBanner))]
+        [Authorize(Roles = Roles.PrimeAdministrant)]
```

Comment:
Prime administrant isn't intended to be used as a permission level like this; its primarily used in the frontend for routing purposes. This should be either Clerk, Analyst, or Manager depending on what the business wants.